apps/src/p5lab/AnimationPicker/AnimationPreview.jsx

```diff
@@ -48,6 +49,25 @@ export default class AnimationPreview extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
For posterity, there's some similar stop/start animation management in the `UNSAFE_componentWillReceiveProps` that I do not fully understand. This new code and the existing code seem not to conflict in meaningful ways as far as I can tell, but seems plausible some simplification/clean up could be done here.

Reply:
Thanks! Just logging that https://github.com/code-dot-org/code-dot-org/pull/42220 updated the deprecated lifecycle methods `componentWillReceiveProps`and `componentWillMount` with the prefix `UNSAFE_`.